docs/getting-started/installation-and-upgrade/installation-references/tls-settings.md

```diff
@@ -23,3 +23,75 @@ The default TLS configuration only accepts TLS 1.2 and secure TLS cipher suites.
+
+## Agent TLS Enforcement
+
+The `agent-tls-mode` setting controls how Rancher's agents (including cluster-agent, fleet-agent, and system-agent) evaluate Rancher's certificate.
+
+When set to `strict`, Rancher's agents will only trust certificates generated by the Certifcate Authority contained in the `cacerts` setting. 
+When set to `system-store`, Rancher's agents will trust any certificate generated by a public Certificate Authority contained in the OS' trust store (including Authorities such as Let's Encrypt). This can be a security risk, since any certificate generated by these authorities will be viewed as valid.
+
+While the `strict` option has a higher level of security, it requires Rancher to have access to the CA which generated the certificate visible to the agents. In the case of certain certificate configurations (notably, external certificates), this is not automatic, and extra configuration will be needed. See the [installation guide](../install-upgrade-on-a-kubernetes-cluster/install-upgrade-on-a-kubernetes-cluster.md#3-choose-your-ssl-configuration) for more information on which scenarios require extra configuration.
+
+This setting defaults to `strict` on new installs of Rancher (starting with v2.9.0), but is set as `system-store` for users installing/upgrading from a prior version.
+
+### Preparing for the Setting Change
+
+Each cluster contains a condition in the status field called `AgentTlsStrictCheck`. If this value is set to true, this indicates the agents for the cluster are ready to operate using the `strict` mode. You can manually inpsect each cluster to see if they are ready using the Rancher UI, or a Kubectl command like the following:
+
+```bash
+## the below command skips ouputs $CLUSTER_NAME,$STATUS for all non-local clusters
+kubectl get cluster.management.cattle.io -o jsonpath='{range .items[?(@.metadata.name!="local")]}{.metadata.name},{.status.conditions[?(@.type=="AgentTlsStrictCheck")].status}{"\n"}{end}'
+```
+
+### Changing the Setting
+
+You can change the setting using the Rancher UI or the `agentTLSMode` [helm chart option](./helm-chart-options.md).
+
+:::note
+
+If the value is set through the helm chart, it must be changed through the helm chart.
+
+:::
+
+:::warning
+
+Depending on your cert setup, additional action (such as uploading the Certificate Authority which signed your certs) may be required. Review the [installation guide](../install-upgrade-on-a-kubernetes-cluster/install-upgrade-on-a-kubernetes-cluster.md#3-choose-your-ssl-configuration) before changing the setting to see if any additional requirements apply to your setup.
+
+:::
+
+To change the option through the UI by navating to the "Global Settings" page, and looking for the `agent-tls-mode` setting near the bottom of the page. When changing the setting through the UI, Rancher will check to ensure that all downstream clusters have the status check set to `true`, in order to avoid outages from a certificate mismatch.
```

Comment:
```suggestion
To change the setting's value through the UI, navigate to the "Global Settings" page and find the `agent-tls-mode` setting near the bottom. When asked to modify the value, Rancher will check that all downstream clusters have the condition `AgentTlsStrictCheck` set to `True`, in order to prevent outages from a certificate mismatch.
```